cmd/analyzer/analyzer.go

```diff
@@ -56,6 +57,7 @@ func parseFlags(version string) cliFlags {
 	fs.BoolVar(&cli.mistJson, "j", false, "Print application info as json")
 
 	fs.BoolVar(&cli.enableStreamHealth, "enable-stream-health", true, "Whether to enable the stream health services and API")
+	fs.BoolVar(&cli.disableBigQuery, "disable-bigquery", true, "Whether to disable the BigQuery-based views and usage APIs")
```

Comment:
Can we make it enabled by default as well? It will annoy the local development a lot if we always need to remember this flag.

Reply:
Oh oops yes I meant to make it `false` by default

Reply:
But I had to change it from `enable-bigquery` to `disable-bigquery` to accommodate Catalyst which can't do `-foo=false` flags

Reply:
hmm could be the case to invert the stream health one then. we might want to not need a rabbitmq for catalyst as well? (tho other services will need to make it optional for that to happen)